fr/mail/chrome/messenger/pbValidate.dtd

```diff
@@ -49,7 +49,7 @@
 
 <!ENTITY enterLicenseTitle.style "">
 <!ENTITY invalidLicenseVersion.style "">
-<!ENTITY validationTitle.style "font-size: 10px;">
+<!ENTITY validationTitle.style "Taille de la police: 10px;">
```

Comment:
WOAH ! Il ne faut SURTOUT pas localiser cette entité car c'est un style CSS et elle DOIT être préservée comme `font-size: 10px;` Merci également d'insérer une ligne de commentaire juste devant: `# do NOT localize entity validationTitle.style`